datajoint/migrate.py

```diff
@@ -0,0 +1,157 @@
+import datajoint as dj
+from pathlib import Path
+import re
+from .utils import user_choice
+
+
+def migrate_dj011_external_blob_storage_to_dj012(migration_schema, store):
```

Comment:
add a doc string describing purpose and input arguments

Reply:
Updated